### DISCUSSION TITLE
Maker Protocol, Maker DAO and DAI

### DISCUSSION THREAD

**Discussion by mgroovyank** [Cyfrin/foundry-full-course-cu#2169] [Show and tell]
Hi everyone! While going through **Introduction to Stablecoins** section of the course, I was introduced to the story of Maker DAO. I was intrigued to understand the working of Maker Protocol. Hence, I decided to read about it myself and I've created a medium post that explains Maker DAO effectively for a beginner. After my research, these concepts became crystal clear to me. You can also have a look at it and see if it helps you as well. Please provide any feedback for improvements. 

Here is the link for the story: [A comprehensive Guide to Maker DAO](https://medium.com/@mayankchhipa007/a-comprehensive-guide-to-maker-protocol-maker-dao-and-dai-stablecoin-6aed7fabeb17)

**Comment by EngrPips** [comment]
Hello @mgroovyank, Thanks for sharing with us this resources and kudos to you for the hardwork.
